src/components/SignUpForm/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./styles.scss";
+
+export default function Form() {
```

Comment:
```suggestion
export default function SignUpForm() {
```

For a better understanding, we should not name the component and file name differently.